nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -284,27 +284,22 @@ public void createNamespace(Namespace namespace, Map<String, String> metadata) {
     return ImmutableMap.of();
   }
 
-  /**
```

Comment:
Does it make sense to add a short comment about the reasoning why Nessie doesn't support `dropNamespace` operation in Iceberg?

Reply:
I would be fine with just "not supported by Nessie backed catalogs" or something